gen/definitions/snmp_view.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: SNMP View
+path: Cisco-IOS-XR-um-snmp-server-cfg:snmp-server/views/view[view-name-%s]/mib-view-families/mib-view-family[mib-view-family-name=%s]
```

Comment:
We want to manage a "view" here, so the path should be `Cisco-IOS-XR-um-snmp-server-cfg:snmp-server/views/view[view-name-%s]`. `mib-view-families/mib-view-family` would then become a nested list.